react-file-uploads/client/app.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
+    /**
+     * - prevent the browser from performing default form submission behavior
+     * - create a new FormData object, passing in the event.target
+     * - send a POST request to /api/uploads with your FormData object as the body
+     *   no headers or additional fetch configuration is needed, just method and body
+     * - log the response body to the console
+     * - reset the form
+     * - catch any promise rejection
+     */
+
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const init = {
+      method: 'POST',
+      body: formData
+    };
+    fetch('/api/uploads', init)
+      .then(res => console.log(res))
```

Comment:
this is just the Response object. the back end is programmed to tell you information about the uploaded file.


```js
fetch('/api/uploads', init)
  .then(res => res.json())
  .then(body => console.log(body))
  .catch(err => console.error(err))
```